### ISSUE TITLE
Swift 1.2 merge

### ISSUE THREAD

**Issue by AnthonyMDev** [drmohundro/SWXMLHash#27]
When is the Xcode 6.3 branch going to be merged into master and pushed to Cocoapods?

I'm waiting on this right now! :) Thanks.

**Comment by drmohundro** [comment]
I'm planning to do that tonight! :)

**Comment by AnthonyMDev** [comment]
Great thanks!

**Comment by drmohundro** [comment]
Hi @AnthonyMDev, I just pushed the 1.0.0 pod to trunk, so you should see it soon - that's Xcode 6.3 and Swift 1.2 support.
